vr/common/utils.py

```diff
@@ -140,6 +141,7 @@ def parse_redis_url(url):
     Given a url like redis://localhost:6379/0, return a dict with host, port,
     and db members.
     """
+    warnings.warn("Use redis.StrictRedis.from_url instead", DeprecationWarning)
```

Comment:
This call should indicate `stacklevel=2` so the caller of `parse_` is implicated in the deprecated usage.